### DISCUSSION TITLE
Convert pro account into free team plan losing features

### DISCUSSION THREAD

**Discussion by kirkcola** [community/community#23315] [New to GitHub]
<p>Hi,</p>
<p>We have a monthly paid Pro account with some private repos, which has many great features for collaboration like protected branches and required reviewers.</p>
<p>After I converted the account into an organization account, those features are no longer available and now require a paid Team plan, since the owner account of the repo has already had a Pro plan subscription, I wonder why we instead losing some features, and how to fix it, thank you.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hello <a class="mention" href="/kirkcola">@kirkcola</a>,</p>
<p>Personal accounts and organization accounts are entirely separate, so unfortunately the features and benefits available to a personal account don’t carry over to an organization. (The reverse is also true; organization features and benefits don’t apply to the personal accounts of its owners and members.) Access to protected branches and required reviewers isn’t available on GitHub Free, so you’d need to <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/upgrading-your-github-subscription#upgrading-your-organizations-subscription" rel="nofollow noopener">upgrade your organization to GitHub Team</a> in order to use them.</p>

**Comment by kirkcola** [comment]
<p>Thanks for the help!</p>
